app/assets/javascripts/application.js

```diff
@@ -60,3 +62,16 @@ var renderChart = function(app, token) {
       .call(xAxis);
   });
 }
+
+var renderTable = function(event_sources) {
```

Comment:
'renderTable' was used before it was defined.<br>Identifier 'event_sources' is not in camel case.